src/senaite/core/upgrade/v02_00_000.py

```diff
@@ -176,8 +175,6 @@ def upgrade(tool):
     _run_import_step(
         portal, "componentregistry", profile="profile-bika.lims:default")
 
-    add_dexterity_setup_items(portal)
```

Comment:
I guess we should remove all upgrade steps before 2.3 or even 2.4 (in another PR), because afaik they have other compatibility issues as well.